book/src/user/shielded-scan.md

```diff
@@ -9,28 +9,82 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
 
 Zebra's shielded scanning feature has known security issues. It is for experimental use only.
 
-Do not use regular or sensitive viewing keys with Zebra's experimental scanning feature. Do not use this
-feature on a shared machine. We suggest generating new keys for experimental use.
+Do not use regular or sensitive viewing keys with Zebra's experimental scanning feature. Do not use this feature on a shared machine. We suggest generating new keys for experimental use.
 
 ## Build & Install
 
-Using the `shielded-scan` feature. TODO: add examples, document the feature in zebrad/src/lib.rs.
+- Clone a Zebra version greater than or equal to [1.5.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.5.0) or use the `main` branch to get the latest features.
+
+```bash
+cargo install --features shielded-scan --locked --git https://github.com/ZcashFoundation/zebra zebrad
+```
+Zebra binary will be at `~/.cargo/bin/zebrad`.
+
+- Generate a configuration file with the default settings:
+
+```bash
+zebrad generate -o ~/.config/zebrad.toml
+```
 
 ## Configuration
 
 In `zebrad.toml`, use:
 - the `[shielded_scan]` table for database settings, and
 - the `[shielded_scan.sapling_keys_to_scan]` table for diversifiable full viewing keys.
 
-TODO: add a definition for DFVK, link to its format, and add examples and links to keys and database settings.
+Sapling diversifiable/extended full viewing keys strings start with `zxviews` as described in [ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
+
+For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/z/all) use:
+
+```toml
+[shielded_scan.sapling_keys_to_scan]
+"zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz" = 1
+```
 
 ## Running Sapling Scanning
 
-Launch Zebra and wait for 12-24 hours.
```

Comment:
It's helpful to tell users they will need to wait a day or so for full scanning results.